src/sharp_plugin.c

```diff
@@ -207,6 +208,24 @@ ncclResult_t ncclSharpListen(int dev, void* opaqueHandle, void** listenComm) {
 ncclResult_t ncclSharpConnect(void* handles[], int nranks, int rank, void* listenComm, void** collComm) {
   struct ncclSharpListenComm* lComm = (struct ncclSharpListenComm*)listenComm;
   struct ncclSharpCollComm* cComm;
+  char *useSharp;
+
+  if(nranks < ncclParamSharpGroupSizeThresh()) {
+    if(rank == 0) {
+      WARN("Group size:%d is less than threshold:%d. fallback to non-sharp",
```

Comment:
Maybe we should we be consistent with using "SHARP" in the INFO/WARN messages so a simpe grep will find them